doc-deploy-stable/action.yml

```
@@ -191,10 +191,11 @@ runs:
         tag=${{ env.REF_NAME }}
         version=$( echo $tag | cut -d 'v' -f 2)
 
-        if [[ $version =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$ ]]; then
+        if [[ $version =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))?$ ]]; then
```

```suggestion
        if [[ $version =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(\..*)?$ ]]; then
```
Another option is to align this with the next regex... up to you @ansys/pyansys-core - I just wanted to remain as less intrusive as possible

---

I would stick to the first one just to avoid any unknown issues caused by suffix tags. Just wondering how the logic would behave.

---

+1